api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -104,6 +105,20 @@ message JwtProvider {
   //
   repeated string audiences = 2;
 
+  // Restrict the `subjects <https://tools.ietf.org/html/rfc7519#section-4.1.2>`_
+  // that the JwtProvider can assert. For instance, this could implement JWT-SVID
+  // `subject restrictions <https://github.com/spiffe/spiffe/blob/main/standards/JWT-SVID.md#31-subject>`_.
+  // If not specified, will not check subjects in the token.
+  //
+  // Example:
+  //
+  // .. code-block:: yaml
+  //
+  //     subjects:
+  //       prefix: spiffe://spiffe.example.com/
+  //
+  type.matcher.v3.StringMatcher subjects = 19;
```

Comment:
This makes sense to me from an API perspective. @lizan ?

Reply:
Yes this makes sense.